nbconvert/exporters/webpdf.py

```diff
@@ -65,40 +68,44 @@ def _template_name_default(self):
         """,
     ).tag(config=True)
 
-    def _check_launch_reqs(self):
-        try:
-            from pyppeteer import launch  # type: ignore[import]
-            from pyppeteer.util import check_chromium  # type:ignore
-        except ModuleNotFoundError as e:
-            msg = (
-                "Pyppeteer is not installed to support Web PDF conversion. "
-                "Please install `nbconvert[webpdf]` to enable."
-            )
-            raise RuntimeError(msg) from e
-        if not self.allow_chromium_download and not check_chromium():
-            msg = (
-                "No suitable chromium executable found on the system. "
-                "Please use '--allow-chromium-download' to allow downloading one."
-            )
-            raise RuntimeError(msg)
-        return launch
-
-    def run_pyppeteer(self, html):
-        """Run pyppeteer."""
+    def run_playwright(self, html):
+        """Run playwright."""
 
         async def main(temp_file):
-            """Run main pyppeteer script."""
+            """Run main playwright script."""
             args = ["--no-sandbox"] if self.disable_sandbox else []
-            browser = await self._check_launch_reqs()(
-                handleSIGINT=False, handleSIGTERM=False, handleSIGHUP=False, args=args
-            )
-            page = await browser.newPage()
-            await page.emulateMedia("print")
```

Comment:
It seems that that option was lost in the upgrade process. As the option is [available in playwright](https://playwright.dev/python/docs/api/class-page#page-emulate-media), I guess it will be better to restore it to increase backward compatibility.

cc @brichet

Reply:
Thanks @fcollonval. I opened https://github.com/jupyter/nbconvert/pull/2022